internal/gok/overwrite.go

```diff
@@ -72,19 +74,35 @@ func (r *overwriteImplConfig) run(ctx context.Context, args []string, stdout, st
 		cfg.InternalCompatibilityFlags = &config.InternalCompatibilityFlags{}
 	}
 
+	if r.full != "" && r.gaf != "" {
+		return fmt.Errorf("cannot specify both --%s and --%s", "full", "gaf")
```

Comment:
No need for the %s placeholders here: 

return fmt.Errorf("cannot specify both --full and --gaf")